Defs/PawnKindDefs/PawnKinds_CE.xml

```diff
@@ -142,6 +142,8 @@
     <weaponTags>
       <li>Gun</li>
     </weaponTags>
+    <combatEnhancingDrugsChance>0.5</combatEnhancingDrugsChance>
```

Comment:
I think this drug would probably be more useful for melee or other close-range (grenadiers?) pawns rather than ranged pawns.

Reply:
It certainly is better suited for melee pawns but the ranged units can also benefit from it.